encoding/da.go

```diff
@@ -354,6 +357,44 @@ func MakeBlobCanonical(blobBytes []byte) (*kzg4844.Blob, error) {
 	return &blob, nil
 }
 
+// BytesFromBlobCanonical converts the canonical blob representation into the raw blob data
+func BytesFromBlobCanonical(blob *kzg4844.Blob) [126976]byte {
+	var blobBytes [126976]byte
+	for from := 0; from < len(blob); from += 32 {
+		copy(blobBytes[from/32*31:], blob[from+1:from+32])
+	}
+	return blobBytes
+}
+
+// DecompressScrollBatchBytes decompresses the given bytes into scroll batch bytes
+func DecompressScrollBatchBytes(compressedBytes []byte) ([]byte, error) {
```

Comment:
The naming here is a bit confusing. The `blob` bytes are decompressed to get `batch` bytes. But `DecompressScrollBatchBytes` sounds as if we are decompressing the batch bytes. A nit-pick.